activejob/test/cases/callbacks_test.rb

```diff
@@ -27,7 +27,10 @@ class CallbacksTest < ActiveSupport::TestCase
   test "#enqueue returns false when before_enqueue aborts callback chain and return_false_on_aborted_enqueue = true" do
     prev = ActiveJob::Base.return_false_on_aborted_enqueue
     ActiveJob::Base.return_false_on_aborted_enqueue = true
-    assert_equal false, AbortBeforeEnqueueJob.new.enqueue
+
+    ActiveSupport::Deprecation.silence do
```

Comment:
Deprecation triggered is related to the change I made, thought the test isn't related, so I'm just silencing deprecation